lib/pdfjs_viewer-rails.rb

```diff
@@ -7,7 +7,7 @@ class Engine < ::Rails::Engine
       isolate_namespace PdfjsViewer
 
       initializer 'pdfjs_viewer-rails.load_static_assets' do |app|
-        app.middleware.unshift ::ActionDispatch::Static, "#{root}/public"
+        app.middleware.insert_before(::ActionDispatch::Static, ::ActionDispatch::Static, "#{root}/public")
```

Comment:
Is there no other way to insert something at the beginning? What if `ActionDispatch::Static` is not part of the production middleware stack?